projects/storefrontstyles/scss/components/misc/chat-messaging/_avatar.scss

```diff
@@ -0,0 +1,18 @@
+%cx-avatar {
+  .cx-avatar {
+    height: 4.375rem;
+    min-width: 4.375rem;
+    background-color: #fff1f1;
+    border-radius: 50%;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    border: 1px solid;
+    border-color: #d3d6db;
```

Comment:
this. can be removed and consolidated. in. the line above

Reply:
got it